components/AddWorkoutForm.tsx

```diff
@@ -0,0 +1,78 @@
+import { View, StyleSheet } from 'react-native';
+import { Button, Overlay, Text, Input, Divider, ListItem } from '@rneui/themed';
+import { useState } from 'react';
+import Ionicons from '@expo/vector-icons/Ionicons';
+import { capitaliseWords } from '@/hooks/utils';
+
+interface Props {
+  visible: boolean;
+  handleToggleOverlay: () => void;
+}
+
+export const AddWorkoutFormOverlay = ({ visible, handleToggleOverlay }: Props) => {
+  const [workoutName, setWorkoutName] = useState<string>('');
```

Comment:
This is fine for now. But imo, you're going to want to invest in React-hook-form sooner rather than later. Will make managing this sort of stuff a bit easier.